tests/conftest.py

```diff
@@ -40,3 +41,13 @@ def isinstance_test(self, py_input):
                 return self.validator.isinstance_python(py_input)
 
     return CustomSchemaValidator
+
+
+if sys.platform == 'emscripten':
```

Comment:
Hopefully we can figure out a better way to deal with fixtures cuz these are a bit annoying. I wonder if we can build the binary dependencies for the `pytest-benchmark` package for Pyodide.

Reply:
Do you mean [`py-cpuinfo`](https://github.com/workhorsy/py-cpuinfo) ? It doesn't seem to require C/C++ build, the only issue is that it did not upload any wheels to [PyPI](https://pypi.org/project/py-cpuinfo/#files).

Reply:
Can you open an issue there asking them to upload a wheel?

Reply:
https://github.com/workhorsy/py-cpuinfo/issues/163 was opened long time ago.

Reply:
I guess I will add py-cpuinfo to Pyodide. Of course there is absolutely no guarantee it would work even if we had it.

Reply:
we could omit optionally `pytest-benchmark` then skip those tests for when it's not installed, or better provide a fallback `benchmark` fixture which just calls the code.

Reply:
ignore my comment, just looking at the code, looks like you've done exactly that.